src/fileseq/filesequence.py

```diff
@@ -62,7 +62,11 @@ def __init__(self, sequence):
         else:
             self._dir = ''
 
-        self._zfill = sum([PAD_MAP[c] for c in self._pad])
+        printf_len = PRINTF_RE.split(self._pad, 1)
+        if len(printf_len) == 3:
```

Comment:
Yes, definitely more readable. I normally used namedgroups so was struggling with the best way to extract what I needed. Thank you!